README.md

```diff
@@ -30,16 +34,39 @@ Due to the rapid changes being made in the React Native ecosystem, we are not of
 ### Linking the Library
 In order to use Touch ID, you must first link the library to your project.  There's excellent documentation on how to do this in the [React Native Docs](http://facebook.github.io/react-native/docs/linking-libraries-ios.html#content).
 
+Or use the built-in command:
+```shell
+react-native link react-native-touch-id
+```
+
+### Platform Differences
+
+iOS and Android differ slightly in their TouchID authentication.
+
+On Android you can customize the title and color of the pop-up by passing in the **optional config object** with a color and title key to the `authenticate` method. Even if you pass in the config object, iOS **does not** allow you change the color nor the title of the pop-up.
+
+Error handling is also different between the platforms, with iOS currently providing much more descriptive error codes.
+
+On android you will need to add
+```xml
+<uses-permission  android:name="android.permission.USE_FINGERPRINT" />
```

Comment:
I think you do not need it. You already add this permission to the manifest library so it will get merged with your manifest (app).